api.js

```diff
@@ -1,10 +1,33 @@
-const exec = require("child_process").exec;
+const { exec, spawn } = require("child_process");
 const fs = require("fs");
 
 const processes = {};
 
 const options = { maxBuffer: 1024 * 1000000 };
 
+let longRuningProcesses = {};
+
+const startLongRuningProcess = (script) => {
+  try {
+    const commandProcess = spawn('npm', ['run', script]);
```

Comment:
if we're not gonna show the output in the terminal, is there a reason why you chose sync over exec?

Reply:
i've used spawn because of https://stackoverflow.com/a/10232330/7091035